pkg/term/tc_other.go

```diff
@@ -1,21 +1,21 @@
 // +build !windows
-// +build !linux !cgo
-// +build !linux !ppc64le
```

Comment:
@tiborvass @clnperez I think 🤞 these are right but would appreciate the double check. We still need this file because tc_solaris defines tcget/sets, but it should include the linux, cgo case.